source/install-using-precompiled-files/index.html.md.erb

```diff
@@ -40,7 +40,7 @@ You’ll not be able to:
         <title>Example - GOV.UK</title>
         <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
         <link rel="manifest" href="/assets/manifest.json">
-        <link rel="stylesheet" href="/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css">
+        <link rel="stylesheet" href="https://design-system.service.gov.uk/stylesheets/govuk-frontend-<VERSION-NUMBER>.min.css">
```

Comment:
What's this change?

Reply:
In case you're still curious: this got caught in the crossfire when I replaced the old Design System links in the migration tables and searched for `"href="styles` to replace with `"href="[DS url]/styles`.